attributes/default.rb

```diff
@@ -31,6 +31,14 @@
 # Location for Prometheus pre-compiiled binary.
 default['prometheus']['binary']['url']                                                    = ''
 
+# Prometheus job configuration chef template name.
+default['prometheus']['job_config_template_name']                                         = 'prometheus.conf.erb'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.